models/safe/zksync/safe_zksync_transactions.sql

```diff
@@ -0,0 +1,112 @@
+{{ 
+    config(
+        materialized='incremental',
+        
+        alias = 'transactions',
+        partition_by = ['block_month'],
+        unique_key = ['block_date', 'tx_hash', 'trace_address'], 
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["zksync"]\',
```

Comment:
just double checking you want to expose this on data explorer, rather than just the cross-chain one above?